scripts/trigger-pipeline.sh

```diff
@@ -19,18 +19,28 @@ cat > /tmp/swissknife/trigger_pipeline.sh <<'EOF'
   export vcs_type="$1";
   export username="$2";
   export reponame="$3";
-  export branch="$4";
-  export params="$5";
+  if [[ $4 == "-t" ]]; then
+    export tag="$5";
+    export params="$6";
+    export body='{
+      "tag": "'$tag'",
+      "parameters": '"$params"'
+    }';
```

Comment:
Went with the approach where we keep the function call the same for a `branch` so that it's backwards compatible and use `-t` with a `tag`. Considered using `getopts`, but since this was a single param figured this was sufficient